ceph-build-pull-requests/build/build

```diff
@@ -29,3 +29,14 @@ for dir in `find . -maxdepth 1 -path ./.git -prune -o -type d -print`; do
         venv/bin/jenkins-jobs test $definitions_dir -o /tmp/output
     fi
 done
+
+# Install Ansible
+if ! venv/bin/pip install --find-links="file://$PIP_SDIST_INDEX" --no-index ansible; then
+    venv/bin/pip install --download-directory="$PIP_SDIST_INDEX" ansible
+    venv/bin/pip install --find-links="file://$PIP_SDIST_INDEX" --no-index ansible
+fi
+
+# Syntax-check each Ansible playbook
+for playbook in ansible/*.yml; do
```

Comment:
this is probably a bit overkill, but would you mind adding the `$WORKSPACE` variable here so that it reads as a full path? Like:

    for playbook in "$WORKSPACE/ansible/*.yml"; do